panther_analysis_tool/main.py

```diff
@@ -580,7 +575,7 @@ def publish_release(args: argparse.Namespace) -> Tuple[int, str]:
         "Authorization": f"token {api_token}",
     }
     # check this tag doesn't already exist
-    response = requests.get(release_url + f"/tags/{args.github_tag}", headers=headers)
+    response = requests.get(release_url + f"/tags/{args.github_tag}", headers=headers, timeout=10)
```

Comment:
How do we know that 10 is the right value here?

Reply:
It's just an arbitrary value. Five would probably suffice but either should be sufficient.